_data/navigation.yml

```
@@ -62,3 +53,17 @@
       url: /contact/team
     - title: Alumni
       url: /contact/alumni
+- title: "Archive"
+  subitems:
+    - title: RSE Call for proposals 2025
+      url: /collaboration/RSEtime/2025
+    - title: CMI-RSE Projects
```

Could add in the 2024 call to the archive as it too has passed.

```suggestion
    - title: RSE Call for Proposals 2024
      url: /collaboration/RSEtime/2024
    - title: CMI-RSE Projects
```